plugins/woocommerce/tests/e2e-pw/playwright.config.js

```diff
@@ -11,7 +11,7 @@ const {
 const config = {
 	timeout: DEFAULT_TIMEOUT_OVERRIDE
 		? Number( DEFAULT_TIMEOUT_OVERRIDE )
-		: 90 * 1000,
+		: 150 * 1000,
```

Comment:
There was no problem running tests locally, but running on CI, tests were timing out.  Bumping this by 60 seconds seems to have resolved the issue on CI (so as discussed, we may need to adjust for daily/release testing)